themes/default/content/solutions/ai.md

```diff
@@ -0,0 +1,59 @@
+---
+title: AI Infrastructure at Any Scale
+meta_desc: Spin up AI infrastructure in any cloud using any programming language
+
+type: page
+layout: ai-solutions
+
+overview:
+    title: Scale AI Infrastructure using Python
+    description: |
+        Pulumi makes it easy to provision, scale, and manage artificial intelligence (AI) and machine learning (ML) infrastructure across any cloud. You can use standard programming languages, like Python, to write infrastructure as code (IaC) programs that automate the infrastructure for your data pipelines, model training clusters, and backend services. Pulumi eliminates the silos between infrastructure and data science teams, allowing you to get to market faster. 
```

Comment:
Workshopped the text a bit and came up with the following. Personally I believe it tidys up the language feeling less rushed and more polished.

Thoughs?

```suggestion
        Pulumi streamlines provisioning, scaling, and management of artificial intelligence (AI) and machine learning (ML) infrastructure across any cloud. Use familiar programming languages, including Python, to develop your infrastructure as code (IaC), automate your data pipelines, model training clusters, and backend services. Pulumi dissolves the barriers between infrastructure and data science teams, accelerating your time to market.
```